e2e_samples/parking_sensors/scripts/configure_databricks.sh

```diff
@@ -135,7 +143,7 @@ databricks libraries install --json @$json_file
 
 # Creates a Job to setup workspace
 log "Creating a job to setup the workspace..."
-notebook_path="${databricks_folder_name}/00_setup.py"
+notebook_path="${DATABRICKS_RELEASE_FOLDER}/00_setup.py"
```

Comment:
Do we need to remove the `.py` from this notebook_path like we did above? (I am having region problems with deploy this AM...so I can't test yet)